.gitignore

```diff
@@ -12,3 +12,4 @@ MANIFEST
 coverage.xml
 .*.sw?
 .idea/
+pip/_vendor/msgpack/*.so
```

Comment:
I kind of feel like instead of adding these to ``.gitignore``, it would be better to add a step to the ``tasks/vendoring/__init__.py`` that actually removed them. It might even be able to be just a patch file that just RM's the relevant ``.so``.

I don't have a great reason *why* I think this is better though, other than I like that the output of a clone and the output of ``inv vendoring.update``  would be the same.